### PR TITLE
Chore/update tou

### PR DESCRIPTION
**Pull request by eugbyte**
I decided to convert the word doc into a json file instead of hard coding, given how large the document is.
`heading: [clause1.1, clause1.2, [`clause1.2.1, clause1.2.2], clause 1.3]'

I then recursively generate the ordered list, making use of the existing "ol-nested" css class that will increment the bullet point accordingly.

So far, I have only copied part of the word doc into json.

### CODE REVIEW

File: src/components/terms/legal-list.css [hunk 1]
```diff
@@ -0,0 +1,19 @@
+/* From https://davelane.nz/creating-legal-list-numbering-only-using-css */
```
Comment (Nebulis) [inline src/components/terms/legal-list.css:1]: let's use css modules => https://create-react-app.dev/docs/adding-a-css-modules-stylesheet/

File: src/components/terms/tou.json [hunk 2]
```diff
@@ -0,0 +1,86 @@
+{
+    "General": [
+        "These Terms of Use govern your access to and use of our services, including the application (whether as software or as a website or otherwise), its contents, push notifications and all other accompanying materials as identified in the Schedule below (collectively, the \"Service\").",
```
Comment (Nebulis) [inline src/components/terms/tou.json:3]: bold stuff should be in bold

File: src/components/terms/legal-list.css (hunk 1 above)
Comment (eugbyte) [reply src/components/terms/legal-list.css]: I have deleted the css file as it was uncessary

Comment (eugbyte) [reply src/components/terms/legal-list.css]: reinstated css file

**Review by Nebulis — APPROVED** [review]

**Pull request merged**